app/scripts/modules/amazon/src/loadBalancer/configure/classic/CreateClassicLoadBalancer.tsx

```diff
@@ -232,19 +226,60 @@ export class CreateClassicLoadBalancer extends React.Component<
         closeModal={this.submit}
         submitButtonLabel={forPipelineConfig ? (isNew ? 'Add' : 'Done') : isNew ? 'Create' : 'Update'}
         validate={this.validate}
-        hideSections={hideSections}
-      >
-        <LoadBalancerLocation
-          app={app}
-          isNew={isNew}
-          forPipelineConfig={forPipelineConfig}
-          loadBalancer={loadBalancer}
-        />
-        <SecurityGroups done={true} isNew={isNew} />
-        <Listeners done={true} />
-        <HealthCheck done={true} />
-        <AdvancedSettings done={true} />
-      </WizardModal>
+        render={({ formik, nextIdx, wizard }) => (
+          <>
+            {showLocationSection && (
+              <WizardPage
+                label="Location"
+                wizard={wizard}
+                order={nextIdx()}
+                render={({ innerRef }) => (
+                  <LoadBalancerLocation
+                    app={app}
+                    formik={formik}
+                    isNew={isNew}
+                    forPipelineConfig={forPipelineConfig}
+                    loadBalancer={loadBalancer}
+                    ref={innerRef}
+                  />
+                )}
+              />
+            )}
+
+            {includeSecurityGroups && (
+              <WizardPage
+                label={FirewallLabels.get('Firewall')}
+                wizard={wizard}
+                order={nextIdx()}
+                render={({ innerRef, onLoadingChanged }) => (
```

Comment:
now that i've seen enough of these, did you try just doing something like

```
render={(pageProps) => { <SecurityGroups formik={formik} isNew={isNew} {...pageProps} /> }}
```

Reply:
/shrug you can't spread a `ref` prop, which then leaves only `onLoadingChanged` and it's only used sometimes

Reply:
Maybe you can spread a ref prop.